io.github.Mrmayman.QuantumLauncher.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.Mrmayman.QuantumLauncher
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: quantum-launcher
+finish-args:
+  # Needs to talk to the network:
+  - --share=network
+  # Wayland access
+  - --socket=wayland
+  # X11 + XShm access, fallback-x11 wont work in wayland
+  - --share=ipc
+  - --socket=x11
```

Comment:
it's either wayland+fallback-x11 or only x11

Reply:
then make it `x11` only.